plugins/katello/nightly/advanced/http_proxies.md

```diff
@@ -62,17 +65,17 @@ To set the global default:
 
 ![Settings Content tab.](/plugins/katello/{{ page.version }}/advanced/settings_content.png)
 
-Note that selecting different http proxies will trigger updates to repositories that have the HTTP proxy policy set to the the global default. Remote proxy settings will be updated asynchronously in a task.
+Note that selecting different HTTP proxies will trigger updates to repositories that have the HTTP proxy policy set to the global default. Remote proxy settings will be updated asynchronously in a task.
 
 ## Bulk Applying HTTP Proxy Policies and HTTP Proxy Selection
 
-HTTP proxy policies and HTTP proxy selection can be selected for one or more products. the policies and proxy selection propogates to all repositories contain in the products.
+HTTP proxy policies and HTTP proxy selection can be selected for one or more products. The policies and proxy selection propogates to all repositories contain in the products.
 
 ### Selecting the Global Default Proxy
 
 To apply the global default proxy policy to one or more products:
   * navigate to: Content > Products
-  * Select one or more products by clicking the row checkbox for each product you want to assign the http policy for
+  * Select one or more products by clicking the row checkbox for each product you want to assign the HTTP policy for
```

Comment:
s/clicking the row checkbox/selecting the row check box/  {click buttons, select boxes}
Sentence is quite long and you already have "oneor more products" above.
Would this do: Select the row check box for each product you want to assign the HTTP policy to.